extension/scripts/features/revive-request/ttReviveRequest.js

```diff
@@ -164,16 +164,16 @@
 				}
 			} else if (provider === "hela") {
 				const response = await fetchData("hela", {
-					section: "api/request",
+					section: "hela/revive",
 					method: "POST",
-					body: { userID: id, userName: name, factionName: faction, source, country },
+					body: { TornID: ('' + id), Username: name, Source: source},

```

Comment:
Use `.toString()` for number to string conversion.

Reply:
Done @Sashank999